src/pages/example-1/Example1.tsx

```diff
@@ -1,13 +1,23 @@
-import React from 'react';
 import './Example1.css';
 import Table from '../../components/table/Table';
 import { useNavigate } from 'react-router-dom';
-import { certificates } from './certificateMockData';
+import { certificates } from '../add-certificate/certificateMockData';
 
-const Example2: React.FC = () => {
+const Example1: React.FC = () => {
   const headers = ['Supplier', 'Certificate type', 'Valid from', 'Valid to'];
   const navigate = useNavigate();
 
+  const handleEdit = (index: number) => {
+    const certificateToEdit = certificates[index];
+    navigate('/add-certificate', {
+      state: { editMode: true, certificateData: certificateToEdit, index },
```

Comment:
please do not do this, use route params and send Id via params /add-certificate/5 for example and then in your AddCertificate.tsx you can in useEffect, extract certificate that you need by that id

even better would be if you introduce indexed-db at this point already and store and fetch everything in indexed db